src/transformations/xyzgridshift.cpp

```diff
@@ -0,0 +1,329 @@
+/******************************************************************************
+ * Project:  PROJ
+ * Purpose:  Geocentric translation using a grid
```

Comment:
So, this one is quite close to +proj=deformation as far as I can tell. Is the main difference that the grids used here have adjustments in geocentric space rather than ENU-space as the deformation operation?

Reply:
> Is the main difference that the grids used here have adjustments in geocentric space rather than ENU-space as the deformation operation?

Those are also conceptually different operations. deformation is about velocities and imply to specify a time difference. xyzgridshift is for static transformations.

Reply:
> Those are also conceptually different operations. deformation is about velocities and imply to specify a time difference. xyzgridshift is for static transformations.

Right, yes, there's that of course :-) What I wanted to figure out, is if there's is any sense in sharing code between these two operations. My initial read through made me think of that but perhaps that is not ideal.

Reply:
Some code sharing in 408cb1e

Reply:
Awesome, thanks!